test/integration/pipeline/multisig_tx_pipeline_test.cpp

```diff
@@ -105,14 +82,40 @@ class MstPipelineTest : public AcceptanceFixture {
    */
   auto makeGetPendingTxsQuery(const std::string &creator,
                               const crypto::Keypair &key) {
-    return shared_model::proto::QueryBuilder()
-        .createdTime(iroha::time::now())
-        .creatorAccountId(creator)
-        .queryCounter(1)
-        .getPendingTransactions()
-        .build()
-        .signAndAddSignature(key)
-        .finish();
+    return complete(baseQry(creator).getPendingTransactions(), key);
+  }
+
+  /**
+   * Query validation lambda - check that empty transactions response returned
+   * @param response - query response
+   */
+  static void no_txs_check(const shared_model::proto::QueryResponse &response) {
+    ASSERT_NO_THROW({
+      const auto &pending_tx_resp = boost::apply_visitor(
```

Comment:
It's better to use `boost::get<shared_model::interface::TransactionsResponse>(response.get())` to get the same result, instead of `SpecifiedVisitor`